### DISCUSSION TITLE
onedrive webdav

### DISCUSSION THREAD

**Discussion by riemsesy** [sircharlo/meeting-media-manager#473] [Ideas] [answered]
Is there a way to have M3 sync with onedrive or google drive?

I found this explanation to access webdav on OneDrive 
https://winscp.net/eng/docs/guide_microsoft_onedrive

you have to have knowledge of creating a app password if one is using MFA on the microsoft account. And it needs to support oauth for modern authentication.

just an idea

**Comment by e-Feitosa** [comment]
https://github.com/sircharlo/meeting-media-manager/discussions/184#discussioncomment-3806313

**Comment by mtdvlpr** [comment, marked as answer]
Since neither Google Drive nor One Drive officially support WebDAV, it would be very difficult to accomplish this. A quick search did not yield any promising results I'm afraid.
